### ISSUE TITLE
Pending issues 

### ISSUE THREAD

**Issue by mpudota907** [ibm-cloud-docs/power-iaas#1]
Following are the pending issues before recreating this repo.

Data Center list cut off
From: gcorneau-ibm
The three selection boxes for data center contents is cut off for the list of non-US data centers (in the section Hardware specifications). With the font size on my web browser it shows "Non-US (FRA04, FRA05, LON04, LON06, SY...)" and mouse over doesn't show the entire list.
https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-about-virtual-server#hardware-specifications
Also, shouldn't there be somewhere on this page just a plain list of the all the available data centers?
======================================
x86 VSI communication with a Power Systems Virtual Server instance
From: dcuenca-17
x86 VSI communication with a Power Systems Virtual Server instance - No description provided.
======================================
Misleading statement about ESS and AIX support
From: anjaklama
Hi, i think the below statement is misleading:
"Using the ESS website to download an ISO image
To download an ISO image from the ESS website, complete the following steps. You can open an SF case against AIX support if you run into any issues."
AIX support can help with ANY technical issue the customer encounters AFTER the ISO is downloaded from ESS. For any issues with maintenance contracts, entitlement and downloading the ISO from ESS, customers need to engage the ESS team directly - https://www.ibm.com/servers/eserver/ess/OpenServlet.wss
======================================
Ticket content - Location missing
From: skuronen
Step 11 in https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-ordering-direct-link-connect#help advises clients to add the following information on support ticket:
--
Customer name:
Customer account ID:
Direct Link Connect subnet:
IBM Cloud IP address:
Power Systems Virtual Server network IP address:
IBM Cloud ASN:
Power Systems Virtual Server network ASN:
Power Systems Virtual Server Private Network (subnet) Name (1):
Power Systems Virtual Server Private Network (subnet) Name (2):
Power Systems Virtual Server Private Network (subnet) Name (3):
Should include the DC location, too?
======================================
High Availability for Power VS For IBM i is missing in the documents
From: minimehta
High Availability and Disaster Recovery options in Power Systems Virtual Servers section in Cloud docs mentions only about AIX edition. IBM i and BYOL options are missing.
Also, for AIX, pricing link for PowerHA SystemMirror doesn't have price. It suggests to call the system representative.
Lastly, What are some of the additional requirements to ensure High Availability for Power VS needs to be documented. For e.g. PowerHA license needs to be included in the price; Additional LPARs need to be added.
======================================
Location Network Providers
From: thomasmatter
Under Networking > Direct Link Connect for Power Systems Virtual Servers > Location > Chart> Network Providers
"IBM Power Virtual Server" is listed as a network provider. That is clearly inaccurate.
Further, the networking diagrams for this are overly complicated. As a technical seller for IBM Cloud neither I nor my customers can make sense of the diagrams - there is a lot of noise. For the networking piece of this, for the most part all we need to know is options on how to connect a) Power Colo to customer on prem b) Power Colo to PoP/DC c) Power Colo to another colo
The existing diagrams include so many pieces it's almost impossible to discern which pieces a customer needs to purchase and setup vs which pieces are included. I had a customer look at these and come to me saying "I just want my on prem network to connect to power cloud, we don't have any other IBM Cloud". Even as an IaaS SME it look me a fair while to figure out which diagram was relevant even though I know how to connect.
tl;dr These network diagrams need to be simplified
Thomas Matter
Technical Sales
CSI/COMMS Market NA
